### ISSUE TITLE
Advanced search: enable selection of multiple fields

### ISSUE THREAD

**Issue by sandaionescu** [ncrncornell/ced2ar#24]
It would be nice if more than one field could be selected for the fielded search. For example, a user might not be sure if a certain keyword would be found in the label or the question text, so s/he might want to try for alternatives.

**Comment by CSimmer** [comment]
Does my 13-Mar example in Advanced search: list of fields needs to be repositioned on the page #25
answer your question about multiple field usage?

**Comment by sandaionescu** [comment]
Yes, I see from your example that multiple fields may be selected. Yet I think that is *not* apparent on the page, in addition to the actual fields being easily overlooked. This is from a simple, outside user perspective. In my experience you cannot expect anybody to actually read instructions. People just don't do it.  What they can do has to be intuited from the page layout/design. That's why I made the comment. The options simply did not *jump* at me. I think this comment is most relevant for issue #25, however. I'll copy some of it in there.

**Comment by CSimmer** [comment]
The Issue "enable selection of multiple fields" is not correct.  The page does support selection of multiple fields.

The UI/page changes sandaionescu is requesting have been moved over to issue #25.

Documentation could be improved to show an Advanced Search multi-field search.  Added enhancement label for documentation.
